### PR TITLE
Remove ThaiISOCountrySubdivisionCode_1p0 which is unused and targets wrong namespace

### PR DESCRIPTION
**Pull request by varokas**
The original error here

```
[ERROR] src-resolve: Cannot resolve the name 'ids5ISO316612A:ISOTwoletterCountryCodeContentType' to a(n) 'type definition' component.
  line 478 of file:/home/travis/build/ETDA/etax-schema/schema/XMLSchemaV2/ETDA/data/standard/QualifiedDataType_1p0.xsd
[ERROR] src-resolve: Cannot resolve the name 'ids5ISO316612A:ISOTwoletterCountryCodeContentType' to a(n) 'type definition' component.
  line 503 of file:/home/travis/build/ETDA/etax-schema/schema/XMLSchemaV2/ETDA/data/standard/Invoice_QualifiedDataType_1p0.xsd
```

comes from this one file that targets the wrong namespace. Since this element isn't referenced by any other XSD, I don't know what is the correct namespace it should be. So I simply remove the file. 

See the file change comment for details

### CODE REVIEW

File: schema/XMLSchemaV2/ETDA/codelist/standard/ThaiISOCountrySubdivisionCode_1p0.xsd [hunk 1]
```diff
@@ -1,434 +0,0 @@
...
-
-The limited permissions granted above are perpetual and will not be revoked by UN/CEFACT or its successors or assigns.
-
-This document and the information contained herein is provided on an "AS IS" basis and UN/CEFACT DISCLAIMS ALL WARRANTIES, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO ANY WARRANTY THAT THE USE OF THE INFORMATION HEREIN WILL NOT INFRINGE ANY RIGHTS OR ANY IMPLIED WARRANTIES OF MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE.
--->
-<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema"
-  xmlns:ids5ISO316612A="urn:un:unece:uncefact:identifierlist:standard:ISO:ISOTwo-letterCountryCode:SecondEdition2006"
-  xmlns:ccts="urn:un:unece:uncefact:documentation:standard:CoreComponentsTechnicalSpecification:2"
-  targetNamespace="urn:un:unece:uncefact:identifierlist:standard:ISO:ISOTwo-letterCountryCode:SecondEdition2006"
```
Comment (varokas) [inline schema/XMLSchemaV2/ETDA/codelist/standard/ThaiISOCountrySubdivisionCode_1p0.xsd]: the target namespace shouldn't say "ISOTwo-letterCountryCode:SecondEdition2006"

**Comment by varokas** [comment]
now we have other errors. But it is from 'MessageFunctionCode' is already defined

```
[ERROR] 'MessageFunctionCode' is already defined
  line 28 of file:/home/travis/build/ETDA/etax-schema/schema/XMLSchemaV2/uncefact/codelist/standard/UNECE_MessageFunctionCode_D14A.xsd
[ERROR] (related to above error) the first definition appears here
  line 28 of file:/home/travis/build/ETDA/etax-schema/schema/XMLSchemaV2/ETDA/codelist/standard/ThaiMessageFunctionCode_1p0.xsd
[ERROR] 'MessageFunctionCodeContentType' is already defined
  line 558 of file:/home/travis/build/ETDA/etax-schema/schema/XMLSchemaV2/uncefact/codelist/standard/UNECE_MessageFunctionCode_D14A.xsd
[ERROR] (related to above error) the first definition appears here
  line 34 of file:/home/travis/build/ETDA/etax-schema/schema/XMLSchemaV2/ETDA/codelist/standard/ThaiMessageFunctionCode_1p0.xsd
[ERROR] 'ThaiDocumentNameCodeInvoice' is already defined
  line 51 of file:/home/travis/build/ETDA/etax-schema/schema/XMLSchemaV2/uncefact/codelist/standard/ThaiDocumentNameCode_Invoice_1p0.xsd
[ERROR] (related to above error) the first definition appears here
  line 51 of file:/home/travis/build/ETDA/etax-schema/schema/XMLSchemaV2/ETDA/codelist/standard/ThaiDocumentNameCode_Invoice_1p0.xsd
[ERROR] 'ThaiDocumentNameCodeInvoiceContentType' is already defined
  line 122 of file:/home/travis/build/ETDA/etax-schema/schema/XMLSchemaV2/uncefact/codelist/standard/ThaiDocumentNameCode_Invoice_1p0.xsd
[ERROR] (related to above error) the first definition appears here
  line 57 of file:/home/travis/build/ETDA/etax-schema/schema/XMLSchemaV2/ETDA/codelist/standard/ThaiDocumentNameCode_Invoice_1p0.xsd
```

**Comment by champillon** [comment]
@varokas 
You may removed UNECE_MessageFunctionCode_D14A.xsd from the QualifiedDataType_1p0.xsd then the duplication will be removed

**Comment by champillon** [comment]
@NopponP มันต้อง remove ตัวที่เป็น unece รึป่าวครับ สำหรับ UNECE_MessageFunctionCode_D14A.xsd แล้วใช้ตัวที่เป็น ThaiMessageFunctionCode แทน ???

**Comment by varokas** [comment]
I dont mind if you will nab my change and combine with your pr. Does this change looks good to you ? I can just merge.

**Comment by varokas** [comment]
Lets talk about the dup in another pr to make this separate issue

**Comment by champillon** [comment]
I think we should merge this PR first then do it in another pr coz I've tried to remove UNECE_MessageFunctionCode_D14A.xsd and ThaiISOCountrySubdivisionCode_1p0 then run and found the other error...

somehow solve ThaiISOCountrySubdivisionCode_1p0 problem with this PR first.

**Comment by varokas** [comment]
ok. I guess you meant: LGTM =)

**Comment by varokas** [comment]
This should solve first case of #4

**Pull request merged**